### ISSUE TITLE
Dark Mode

### ISSUE THREAD

**Issue by earth-walker** [pixelfed/pixelfed-rn#35]
Please add dark mode 🖤

**Comment by lesimoes** [comment]
@dansup , I would like to contribute with pixelfed. Could I implement this feature?
What you think at Settings -> Appearance -> Theme (Light, Dark, System Preferences)  


<img src="https://github.com/user-attachments/assets/a02e9373-5469-4f79-b817-0fa86f1b4702" alt="F549ADDB-CCC6-4439-8D6C-3FD9527051DF_1_101_o" width="300"/>

**Comment by Valinwolf** [comment]
+1 came here to request this

**Comment by digital-pet** [comment]
Just grabbed the iOS app off TestFlight and noticed this when it just about burnt my retinas out. With the proliferation of OLED screens, dark mode really is not optional.

**Comment by Justinzobel** [comment]
For both Android and iOS please.

**Comment by pfr-dev** [comment]
Just commenting in support of this. Dark mode is essential.

**Comment by applecargo** [comment]
it is immediately needed- I just cannot look at it. lol

**Comment by pfr-dev** [comment]
> [@dansup](https://github.com/dansup) , I would like to contribute with pixelfed. Could I implement this feature? What you think at Settings -> Appearance -> Theme (Light, Dark, System Preferences)

It's unfortunate that Dan never responded to this. Perhaps just submit a PR with the enhancements and hopefully that'll grab their attention.

**Comment by Kwasow** [comment]
Hi, this is currently being worked on

**Comment by dansup** [comment]
Now available in 1.2.0

**Comment by pfr-dev** [comment]
Yes, got the push notification for update this morning. Was very pleased! Thank you 🫶

**Comment by Zahrun** [comment]
Thank you for addding the dark mode.
Can you please add the option to follow system setting? https://github.com/pixelfed/pixelfed-rn/issues/63
